sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -295,6 +295,41 @@ case class AlterViewAsCommand(
   }
 }
 
+/**
+ * A command for users to get views in the given database.
+ * If a databaseName is not given, the current database will be used.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW VIEWS [(IN|FROM) database_name] [[LIKE] 'identifier_with_wildcards'];
+ * }}}
+ */
+case class ShowViewsCommand(
+    databaseName: Option[String],
```

Comment:
why does this need to be an `Option`? We always pass `Some(ns.head)`, right?

Reply:
Yea, the database name should have been resolved. Thanks! Updated in 7bd9cc662528a7fcfdb1f07b6565a0e6cac8512a.